aptos-move/aptos-vm-types/src/storage.rs

```diff
@@ -353,6 +357,12 @@ impl CheckChangeSet for ChangeSetConfigs {
     fn check_change_set(&self, change_set: &VMChangeSet) -> Result<(), VMStatus> {
         const ERR: StatusCode = StatusCode::STORAGE_WRITE_LIMIT_REACHED;
 
+        if self.max_write_ops_per_transaction != 0
```

Comment:
why != 0?

Reply:
before gas schedule is updated (and gas feature is enabled) the gas parameter will read 0

Reply:
I thought you put u64::MAX there?

Reply:
no see `fn from_gas_params()`